packages/server/src/main/java/software/uncharted/terarium/hmiserver/service/TerariumAssetCloneService.java

```diff
@@ -236,6 +237,9 @@ public Project importProject(final String userId, final String userName, final P
 		// set the current user id
 		projectExport.getProject().setUserId(userId);
 		projectExport.getProject().setUserName(userName);
+		projectExport.getProject().setCreatedOn(new Timestamp(System.currentTimeMillis()));
+		projectExport.getProject().setUpdatedOn(new Timestamp(System.currentTimeMillis()));
+		projectExport.getProject().setPublicProject(false);
```

Comment:
This is technically not saved in the databased and created on the fly when getting project by checking ReBac permissions